library/classes.lisp

```diff
@@ -238,14 +238,14 @@ specify `repr :lisp`."
     (alt ((:f :a) -> (:f :a) -> (:f :a)))
     (empty (:f :a)))
 
-  (define-class (Foldable :t)
+  (define-class (Foldable :container)
```

Comment:
IIRC @eliaslfox and I had some back-and-forth on this style. @eliaslfox, Do you have any objections to this?

I'm personally happy with it.

Reply:
I rewrote this while writing the foldable instance for tree, because I was having trouble keeping track of which type was `:a` and which type was `:b`.

Reply:
I don't mind